packages/cortex-react/src/components/Badge.tsx

```diff
@@ -32,6 +32,12 @@ export const Badge = forwardRef<
   );
 });
 
+function getDisplayValue(value: string | number) {
+  const number = Number(value);
+  if (isNaN(number)) return value;
```

Comment:
Esse isNan tem como finalidade realmente ver se o número é inválido? Ou é mais pra "dar" true quando a gente passa uma string?
Se for o segundo caso imagino que ficaria mais "claro" no código tratar com typeof na minha opinião.
if (typeof value == string) return value
const number = Number...

Reply:
entendi, nesse caso faz sentido 👍🏻